content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/cycle_counts.rst

```diff
@@ -11,101 +11,103 @@ companies use *cycle counts* to keep critical stock levels accurate. Cycle count
-location. Click :guilabel:`Edit` to edit the location settings.
+location.
 
-Under the :guilabel:`Cyclic Inventory` section, locate the :guilabel:`Inventory Frequency (Days)`
-field, which should be set to `0` (if this location has not been edited previously). In this field,
-change the value to whichever number of days is desired.
+Under the :guilabel:`Cyclic Counting` section, locate the :guilabel:`Inventory Frequency (Days)`
+field, which should be set to `0` by default (if this location has not been edited previously). In
+this field, change the value to whichever number of days is desired for the frequency of counts.
 
-.. image:: cycle_counts/cycle-counts-location-frequency.png
+.. image:: cycle_counts/cycle-counts-frequency-value.png
    :align: center
    :alt: Location frequency setting on location.
 
 .. example::
    A location that needs an inventory count every 30 days should have the :guilabel:`Inventory
    Frequency (Days)` value set to `30`.
 
-Once the frequency has been changed to the desired number of days, click :guilabel:`Save` to save
-changes. Now, once an inventory adjustment is applied to this location, the next scheduled count
-date is automatically set, based on the value entered into the :guilabel:`Inventory Frequency
-(Days)` field.
+Now, once an inventory adjustment is applied to this location, the next scheduled count date is
+automatically set, based on the value entered into the :guilabel:`Inventory Frequency (Days)` field.
 
 Count inventory by location
 ===========================
 
 To perform a cycle count for a specific location in the warehouse, navigate to
-:menuselection:`Inventory app --> Operations --> Inventory Adjustments`. This reveals an
+:menuselection:`Inventory app --> Operations --> Physical Inventory`. This reveals an
 :guilabel:`Inventory Adjustments` page containing all products currently in stock, with each product
 listed on its own line.
 
 From this page, the :guilabel:`Filters` and :guilabel:`Group By` buttons (at the top of the page,
-under the :guilabel:`Search...` bar), can be used to select specific locations and perform inventory
-counts.
+to the right of the :guilabel:`Search...` bar), can be used to select specific locations and perform
+inventory counts.
 
-.. image:: cycle_counts/cycle-counts-inventory-adjustments-page.png
-   :align: center
-   :alt: Inventory adjustments page.
+To select a specific location and view all products within that location, click the :guilabel:`⬇️
+(down arrow)` icon, to the right of the :guilabel:`Search...` bar. Then, in the :guilabel:`Group By`
+column, click :guilabel:`Add Custom Group` to reveal a new drop-down menu.
 
-To select a specific location and view all products within that location, click :guilabel:`Group
-By`, then click :guilabel:`Add Custom Group` to reveal a new drop-down menu to the right.
+.. image:: cycle_counts/cycle-counts-filter-menu.png
+   :align: center
+   :alt: Filters and Group By menu on Inventory Adjustments page.
```

Comment:
nitpick: what do you think about retaking this image to include the down arrow icon? I was thinking that it's kind of small and hard for a first time user to find, and it would be helpful to have it pictured. Up to you, though!